pkg/pdf/pdf.go

```diff
@@ -282,6 +284,11 @@ func (s *PdfMaroto) SetAliasNbPages(alias string) {
 	s.Pdf.AliasNbPages(alias)
 }
 
+// SetCompression allows to set/unset compression for a pdf
```

Comment:
Could you explain the default value of this feature?

Reply:
done